src/lib/stack/stack.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, ElementRef, HostBinding, Input, OnChanges, OnInit, Renderer2, SimpleChanges } from '@angular/core';
+import { SizeProp } from '@fortawesome/fontawesome-svg-core';
+
+/**
+ * Fontawesome layers.
+ */
+@Component({
+  selector: 'fa-stack',
+  template: `<ng-content select="fa-icon"></ng-content>`,
+})
+export class FaStackComponent implements OnInit, OnChanges {
+  @Input() size?: SizeProp;
+  @Input() fixedWidth?: boolean;
+
+  @HostBinding('class.fa-fw') get isFixedWidth() { return this.fixedWidth || <any>this.fixedWidth === ''; }
```

Comment:
Please remove support for attributes without values from this PR. While it saves couple of characters for users, the implementation looks quite dirty and I don't think this is a common practice for Angular components to support attributes without values.